<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditChestDialogClass</class>
 <widget class="QDialog" name="EditChestDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Graal Chest</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Chest options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Item:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true"/>
         </item>
         <item>
          <widget class="QComboBox" name="itemComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>greenrupee</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bluerupee</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>redrupee</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bombs</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>darts</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>heart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>glove1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>bomb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>shield</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>sword</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fullheart</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>superbomb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>battleaxe</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>goldensword</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>mirrorshield</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>glove2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>lizardshield</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>lizardsword</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>goldrupee</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fireball</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>fireblast</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nukeshot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>joltbomb</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>spinattack</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Sign Index:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="signIndexEdit">
           <property name="text">
            <string>-1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signsButton">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Signs</string>
           </property>
           <property name="icon">
            <iconset resource="../Resources/MainWindow.qrc">
             <normaloff>:/MainWindow/icons/tinycolor/icons8-sign-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-sign-16.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset resource="../Resources/MainWindow.qrc">
          <normaloff>:/MainWindow/icons/tinycolor/icons8-delete-16.png</normaloff>:/MainWindow/icons/tinycolor/icons8-delete-16.png</iconset>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
